taskcluster/ci/build/kind.yml

```diff
@@ -39,15 +39,13 @@ jobs:
             nightly: true
         run-on-tasks-for: []
         include-nightly-version: true
+        include-shippable-secrets: true
         run:
             gradlew: ["-PcrashReportEnabled=true", "-Ptelemetry=true", "assembleNightly"]
             secrets:
                 - path: .sentry_token
                   key: dsn
                   name: project/mobile/reference-browser/sentry
-                - path: app/src/main/res/values/firebase.xml
```

Comment:
Bug: Let's also remove the section in `build-bundle/kind.yml`: https://github.com/mozilla-mobile/reference-browser/pull/868/files#diff-3f36513a7b9b1a2b73d698e2b0b135e2R44. Otherwise that task will keep failing too.

Reply:
Thanks for catching, fixed!